docs/developer/dependencies.md

```diff
@@ -0,0 +1,58 @@
+# Dependencies
+
+We have a number of external dependencies for both the frontend and backend.
+
+## Updating python requirements
+
+The python requirements are defined in `requirements.in` and `requirements_dev.in` these are used to generate `requirements.txt` and `requirements_dev.txt` using `pip-compile`.
+
+
+### Using pip-compile in docker
+
+(Re)Generate the requirements files:
+
+```
+$ docker run --rm -v $(pwd):/code standards-lab_standards-lab pip-compile
+```
+
+Upgrade the requirements:
+
+```
+$ docker-compose run --rm -v $(pwd):/code standards-lab-web pip-compile --upgrade <selected requirements.in file>
+```
+
+
+### Using pip-compile locally
+
+(Re)Generate the requirements files:
+
+```
+$ pip-compile
```

Comment:
This needs to be run for requirements.txt (the default if no file is specified) and requirements_dev.txt.